docs/docs/constraints.md

```diff
@@ -6,21 +6,21 @@ title: Constraints
 
 Constraints control where apps run to allow optimizing for either fault tolerance (by spreading a task out on multiple nodes) or locality (by running all of an applications tasks on the same node). Constraints have three parts: a field name, an operator, and an optional parameter. The field can be the hostname of the agent node or any attribute of the agent node.
 
-## Fields
+## Field Names
 
-### Hostname field
+### Hostname as field name
 
-The `hostname` field matches the agent node hostnames. See `UNIQUE operator`, below, for a usage example.
+Entering `hostname` as the field name matches the agent node hostnames. See `UNIQUE operator`, below, for a usage example.
 
-`hostname` field supports all operators of Marathon.
+All Marathon operators are supported when the field name is `hostname`. 
 
-### Attribute field
+### Attribute as field name
 
 If the field name is not the agent node hostname, it will be treated as a Mesos agent node attribute. A Mesos agent node attribute allows you to tag an agent node. See `mesos-slave --help` to learn how to set the attributes.
```

Comment:
"If the field name is not the agent node hostname..."

should perhaps be

"If `hostname` is not specified as the field name then the field name is interpreted as a Mesos agent node attribute."